src/views/tabela_exercicios.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bootstrap demo</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+
+  </head>
+<body>
+	<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa" crossorigin="anonymous"></script>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<script>
+		fetch('http://localhost:3000/api/exercicios/')
+			.then(res => res.json())
+			.then(data => {
+		 			 	const nome_exercicios = data.result
+		 			 	console.log(nome_exercicios)
+		 			 	buildTable(nome_exercicios)
+		  				
+					})
+			
+			
+		/*function getFullName(item){
+			return [item.nome_exercicio,item.nome_equipamento,item.nome_dificuldade,item.nome_musculo].join(" ");
+		}*/
+
+		function renderNames(names){
+			const container = document.getElementById('exercicios');
+			container.innerHtml = '';
+				for (const name of names) {
+				  const p = document.createElement('p');
+				  const textNode = document.createTextNode(name);
+				  p.appendChild(textNode);
+				  container.appendChild(p);
+				}
+		}
+
+		function buildTable(data){
+			var table = document.getElementById('myTable');
+
+			for(var i = 0; i<data.length; i++){
+				var row = ` <tr>
+								<td>${data[i].id}</td>
+								<td>${data[i].nome_exercicio}</td>
+								<td>${data[i].nome_equipamento}</td>
+								<td>${data[i].nome_dificuldade}</td>
+								<td>${data[i].nome_musculo}</td>
+							</tr>`
+				table.innerHTML += row
+			}
+		}
```

Comment:
Com innerHTML, com for loop, vai percorrer cada linha e mostrar cada exercício em cada linha e junta se a tabela com id myTable